Description: Link against libvorbis and libogg to fix FTBFS
Author: James Cowgill <james410@cowgill.org.uk>
Bug-Debian: https://bugs.debian.org/769213
Forwarded: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/scons/libs/clam_dependent_libs_checks.py
+++ b/scons/libs/clam_dependent_libs_checks.py
@@ -88,7 +88,7 @@ def test_sndfile( env, conf ) :
 	return True
 
 def test_oggvorbis( env, conf ) :
-	if not conf.CheckPkgConfigFile(['vorbisfile', 'vorbisenc']) :
+	if not conf.CheckPkgConfigFile(['vorbisfile', 'vorbisenc', 'vorbis', 'ogg']) :
 		return False
 	if not conf.CheckLibrarySample('libogg', 'c', None, libogg_test_code ) : return False
 	if not conf.CheckLibrarySample('libvorbis', 'c', None, libvorbis_test_code ) : return False
